jetstream/statistics.py

```diff
@@ -787,3 +791,42 @@ def transform(
                     )
                 )
         return StatisticResultCollection(results)
+
+
+@attr.s(auto_attribs=True)
+class PopulationRatio(Statistic):
```

Comment:
Should this be called `PopulationRatio` or something more general, like `Ratio`?

Reply:
If you want the code to be understandable to data scientists in the future, call it `PopulationRatio` because that's a good description of what's happening here statistically. This [line](https://github.com/mozilla/jetstream/blob/d474dbe2969ce041232d5b92df7c3489ec2ba7ae/jetstream/statistics.py#L820) makes it a population ratio, not a statistic on arbitrary subgroups.

If this repo is not meant for data scientists to contribute to, then feel free to call it something else.

Reply:
`PopulationRatio` it is then. These statistics are definitely more aimed towards DS and most of them got implemented by DS

Reply:
Yep, I'm happy with my vote being outweighed here.